Cake\I18n\Time toQuarter() return Int

jonmc1 opened the pull request:
GitHub Issue #12903

Cast return value of toQuarter() to an int

jonmc1 committed:
Cake\I18n\Time toQuarter() return Int

GitHub Issue #12903

dereuromark commented on src/I18n/Time.php:
```diff
@@ -174,7 +174,7 @@ public function toQuarter($range = false)
     {
         $quarter = ceil($this->format('m') / 3);
```
technically, the cast here is better

jonmc1 committed:
Cake\I18n\Time toQuarter() change location Int cast

GitHub Issue #12903

dereuromark commented:
https://github.com/cakephp/cakephp/blob/master/tests/TestCase/View/Helper/TimeHelperTest.php#L156
These kind of comparisons are the problem of all eval :)

jonmc1 committed:
Cake\I18n\Time toQuarter() update tests

Update test case to check for the same value and type
instead of just the value

GitHub Issue #12903
